temperature/utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import httpx
+from fastapi import HTTPException
+
+from settings import settings
+
+
+WEATHER_URL = (
+    f"http://api.weatherapi.com/v1/current.json?key={settings.WEATHER_TOKEN}"
```

Comment:
Ensure that `settings.WEATHER_TOKEN` is correctly set in your environment or configuration file. Without a valid API key, the request to the weather API will fail.